website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md

```diff
@@ -7,7 +7,7 @@ sidebar_label: "dbt Semantic Layer"
 
 :::info 📌
 
-The dbt Semantic Layer is currently available in Public Preview for multi-tenant dbt Cloud accounts hosted in North America. 
+The dbt Semantic Layer is currently available in Public Preview for multi-tenant dbt Cloud accounts hosted in North America (logging in via https://cloud.getdbt.com/)
```

Comment:
@mirnawong1 can we make this public preview note a reusable so it can remain consistent throughout all pages it is used on? Also, it'll be easier to update it.

Reply:
good callout @runleonarun and i'll create a snippet for it!